components/desktop/hexchat/Makefile

```diff
@@ -56,18 +56,15 @@ CONFIGURE_OPTIONS += -Dwith-python=python-3.9
 CONFIGURE_OPTIONS += -Dwith-lua=false
 CONFIGURE_ENV += AR="/usr/gnu/bin/ar"
 
```

Comment:
Whenever you change a package you'll either need to change/increment COMPONENT_VERSION or in case the package's version didn't change you'll need to introduce/increment COMPONENT_REVISION. Here you'll need to add COMPONENT_REVISION=1 to the Makefile.